internal/db/db_test.go

```diff
@@ -58,3 +61,9 @@ func TestOpen(t *testing.T) {
 		})
 	}
 }
+
+func TestPostgresInfinity(t *testing.T) {
+	assert.Panics(t, func() {
```

Comment:
1. To test that `pq.EnableInfinityTs` has been called.

2. The last sentence of the [`func EnableInfinityTs`](https://pkg.go.dev/github.com/lib/pq#EnableInfinityTs) doc states:
> If EnableInfinityTs is called more than once, it will panic.